### ISSUE TITLE
Custom mapping types support

### ISSUE THREAD

**Issue by amchii** [fcurella/django-fakery#62]
I have a custom field inherits from `models.IntegerField`:
```python
class UnsignedIntegerField(models.IntegerField):
    def db_type(self, connection):
        return "integer UNSIGNED"

    def rel_db_type(self, connection):
        return "integer UNSIGNED"

```
but `django-fakery` treats it as `models.IntegerField` and makes a negative number, then error occured.
So could you make an api to custom mappings types for custom fields😄?

**Comment by fcurella** [comment]
I agree there should be a way to add custom fields.

In the meantime, would subclassing `models.PositiveIntegerField` be an option?

**Comment by amchii** [comment]
> In the meantime, would subclassing `models.PositiveIntegerField` be an option?

Yes, I've thought about it, but legacy code with specified database works fine and `models.PositiveBigIntegerField ` new in `Django 3.1`.

**Comment by fcurella** [comment]
I've merged in support for custom fields in https://github.com/fcurella/django-fakery/pull/63 and released it as v3.1.0. Let me know if you get into any issues :)

**Comment by amchii** [comment]
OK, I will. Thanks!
